README.md

```diff
@@ -283,6 +283,14 @@ Commands:
 
   patch <ID> <FILE>          Apply patch FILE to ID.
 
+  env-get                    Get the complete environment of the cluster master
```

Comment:
Scratch that. I'll make it targetable, but I'm going to skip the `env` alias since this is meant to be lower level.